### PR TITLE
Content for configuration backups for Kafka

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Kafka cluster configuration backup allows the restoration of a Kafka cluster to the previous state after a power off/ on cycle or an incident that has caused the cluster to stop working.

For more info - https://aiven.atlassian.net/browse/HH-1164

### CODE REVIEW

File: docs/products/kafka/concepts/configuration-backup.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* Everything related to Schema Registry (schemas, ids, subjects, compatibility levels).
+* Apache Kafka® Connect configurations.
+
+Benefits of configuration backups
+-------------------------------------
+Some of the key benefits of configuration backups include the following: 
+
+* Configuration backups are automatically enabled and stored in the Cloud storage.
+* Configurations are backed up in 3 hours intervals.
+* It helps with speedy disaster recovery.
```
Comment (HelenMel) [inline docs/products/kafka/concepts/configuration-backup.rst:26]: I would add here `speedy disaster recovery in certain situations*` *In a case when all nods are lost and current configurations are not accessible anymore.

When one or two nodes are lost, usually, we do not lose a configuration. Losing all nodes or brain split could lead to configuration lost, but this is quite rare situation. So "quick re-creation of Apache Kafka® services" is a main use-case here.

Comment (harshini-rangaswamy) [reply docs/products/kafka/concepts/configuration-backup.rst]: Added the info about the nodes lost as a note.

**Comment by dewan-ahmed** [comment]
Hi @harshini-rangaswamy @HelenMel friendly nudge to move this PR forward :)

**Comment by harshini-rangaswamy** [comment]
> Hi @harshini-rangaswamy @HelenMel friendly nudge to move this PR forward :)



> Hi @harshini-rangaswamy @HelenMel friendly nudge to move this PR forward :)

Thanks. There were some issues that needed to be fixed before publishing this content. I should have more updates next week.

**Comment by celestehorgan** [comment]
@harshini-rangaswamy - Any updates on this PR? Do you need help pushing it through?

**Comment by harshini-rangaswamy** [comment]
> @harshini-rangaswamy - Any updates on this PR? Do you need help pushing it through?

Hey @celestehorgan, this feature is currently being rolled out gradually to existing customers. 

> @harshini-rangaswamy - Any updates on this PR? Do you need help pushing it through?

@celestehorgan - This feature is not GA for all customers yet but is enabled for specific existing and new customers. According to the dev team, the GA for all the existing customers requires some manual work and could take a bit longer.

**Comment by harshini-rangaswamy** [comment]
> @harshini-rangaswamy - Any updates on this PR? Do you need help pushing it through?

@celestehorgan - This feature is not GA for all customers yet but is enabled for specific existing and new customers. According to the dev team, the GA for all the existing customers requires some manual work and could take longer. 

Do you think it is a good idea to close this PR and create a new one when the feature is ready?

**Comment by filip-yonov-aiven** [comment]
Hello - backups are enabled for all new services. There is a gradual rollout which requires cluster recycle to get the feature in. @AnatolyPopov can provide more information if need be. We will follow soon with official comms on this.

**Comment by harshini-rangaswamy** [comment]
Closing this pull request due to multiple conflicts. The content has been recreated in this PR - https://github.com/aiven/devportal/pull/1768

**Pull request closed without merging**